### PR TITLE
[EAGLE-791] Refactor eagle module names to be more meaningful

### PR DESCRIPTION
**Pull request by haoch**
Refactor eagle module names to be more meaningful and easier for developer to learn about eagle's code structure.

https://issues.apache.org/jira/browse/EAGLE-791

	[INFO] Eagle::Parent ...................................... SUCCESS [  0.231 s]
	[INFO] Eagle::Core::Parent ................................ SUCCESS [  0.034 s]
	[INFO] Eagle::Core::DataProcess ........................... SUCCESS [  0.119 s]
	[INFO] Eagle::Core::Common ................................ SUCCESS [  0.408 s]
	[INFO] Eagle::Core::Alert ................................. SUCCESS [  0.031 s]
	[INFO] Eagle::Core::Alert::Core ........................... SUCCESS [  0.043 s]
	[INFO] Eagle::Core::Alert::Common ......................... SUCCESS [  0.724 s]
	[INFO] Eagle::Core::Alert::Engine ......................... SUCCESS [  1.924 s]
	[INFO] Eagle::Core::Alert::Metadata ....................... SUCCESS [  0.036 s]
	[INFO] Eagle::Core::Alert::MetadataBase ................... SUCCESS [  0.177 s]
	[INFO] Eagle::Core::Alert::MetadataService ................ SUCCESS [  0.097 s]
	[INFO] Eagle::Core::Alert::Coordinator .................... SUCCESS [  0.766 s]
	[INFO] Eagle::Core::Alert::Assembly ....................... SUCCESS [  0.084 s]
	[INFO] Eagle::Core::Alert::DevTools ....................... SUCCESS [  0.335 s]
	[INFO] Eagle::Core::Alert::ServiceApp ..................... SUCCESS [  0.411 s]
	[INFO] Eagle::Core::Query::Parent ......................... SUCCESS [  0.039 s]
	[INFO] Eagle::Core::Query::AntlrCompiler .................. SUCCESS [  0.133 s]
	[INFO] Eagle::Core::Embed::Parent ......................... SUCCESS [  0.030 s]
	[INFO] Eagle::Core::Embed::EmbedHBase ..................... SUCCESS [  0.147 s]
	[INFO] Eagle::Core::Query::EntityBase ..................... SUCCESS [  1.624 s]
	[INFO] Eagle::Core::Query::QueryBase ...................... SUCCESS [  0.383 s]
	[INFO] Eagle::Core::Query::StorageBase .................... SUCCESS [  0.301 s]
	[INFO] Eagle::Core::Embed::EmbedServer .................... SUCCESS [  0.355 s]
	[INFO] Eagle::Core::Query::ClientBase ..................... SUCCESS [  0.209 s]
	[INFO] Eagle::Core::Query::AuditBase ...................... SUCCESS [  0.143 s]
	[INFO] Eagle::Core::Query::HBaseStorage ................... SUCCESS [  0.463 s]
	[INFO] Eagle::Core::Query::ServiceBase .................... SUCCESS [  0.189 s]
	[INFO] Eagle::Core::Metadata .............................. SUCCESS [  0.021 s]
	[INFO] Eagle::Core::Metadata::Base ........................ SUCCESS [  0.303 s]
	[INFO] Eagle::Core::Alert::Service ........................ SUCCESS [  0.113 s]
	[INFO] Eagle::Core::App::Parent ........................... SUCCESS [  0.050 s]
	[INFO] Eagle::Core::App::Base ............................. SUCCESS [  1.125 s]
	[INFO] Eagle::Core::Alert::AppProvider .................... SUCCESS [  0.134 s]
	[INFO] Eagle::Core::Query::JDBCStorage .................... SUCCESS [  0.524 s]
	[INFO] Eagle::Core::Metric ................................ SUCCESS [  0.123 s]
	[INFO] Eagle::Core::Metadata::JDBC ........................ SUCCESS [  0.265 s]
	[INFO] Eagle::Core::Metadata::MongoDB ..................... SUCCESS [  0.183 s]
	[INFO] Eagle::Core::App::Utils ............................ SUCCESS [  0.358 s]
	[INFO] Eagle::App::Security::Parent ....................... SUCCESS [  0.003 s]
	[INFO] Eagle::App::Security::Common ....................... SUCCESS [  0.004 s]
	[INFO] Eagle::App::Security::HdfsAuditLog ................. SUCCESS [  0.034 s]
	[INFO] Eagle::App::Security::MapRFSAuditLog ............... SUCCESS [  0.002 s]
	[INFO] Eagle::App::JPM::Parent ............................ SUCCESS [  0.001 s]
	[INFO] Eagle::App::JPM::Utils ............................. SUCCESS [  0.312 s]
	[INFO] Eagle::App::Security::HiveSecurity ................. SUCCESS [  0.003 s]
	[INFO] Eagle::App::Security::HiveService .................. SUCCESS [  0.002 s]
	[INFO] Eagle::App::Security::HdfsService .................. SUCCESS [  0.006 s]
	[INFO] Eagle::App::Security::MapRFSService ................ SUCCESS [  0.003 s]
	[INFO] Eagle::App::Security::HdfsAuthLog .................. SUCCESS [  0.002 s]
	[INFO] Eagle::App::Security::HBaseAuditLog ................ SUCCESS [  0.043 s]
	[INFO] Eagle::App::Security::HBaseService ................. SUCCESS [  0.002 s]
	[INFO] Eagle::App::Security::MetricCollection ............. SUCCESS [  0.002 s]
	[INFO] Eagle::App::Security::OozieAuditLog ................ SUCCESS [  0.009 s]
	[INFO] Eagle::App::Security::OozieService ................. SUCCESS [  0.004 s]
	[INFO] Eagle::External::Parent ............................ SUCCESS [  0.001 s]
	[INFO] Eagle::External::Log4jKafkaAppender ................ SUCCESS [  0.024 s]
	[INFO] Eagle::External::KafkaExtension .................... SUCCESS [  0.002 s]
	[INFO] Eagle::Example::Parent ............................. SUCCESS [  0.001 s]
	[INFO] Eagle::Example::AppSample .......................... SUCCESS [  0.014 s]
	[INFO] Eagle::App::GcLogMonitor ........................... SUCCESS [  0.042 s]
	[INFO] Eagle::App::JPM::Entity ............................ SUCCESS [  0.199 s]
	[INFO] Eagle::App::JPM::MRRunningJob ...................... SUCCESS [  0.026 s]
	[INFO] Eagle::App::JPM::MRHistoryJob ...................... SUCCESS [  0.028 s]
	[INFO] Eagle::App::JPM::WebUI ............................. SUCCESS [  0.087 s]
	[INFO] Eagle::App::JPM::JobService ........................ SUCCESS [  0.162 s]
	[INFO] Eagle::App::JPM::JobAggregation .................... SUCCESS [  0.008 s]
	[INFO] Eagle::App::JPM::SparkHistoryJob ................... SUCCESS [  0.008 s]
	[INFO] Eagle::App::JPM::HadoopQueue ....................... SUCCESS [  0.035 s]
	[INFO] Eagle::App::TopologyCheck::Parent .................. SUCCESS [  0.002 s]
	[INFO] Eagle::App::TopologyCheck::Entity .................. SUCCESS [  0.056 s]
	[INFO] Eagle::App::TopologyCheck::AppProvider ............. SUCCESS [  0.010 s]
	[INFO] Eagle::App::HadoopMetric ........................... SUCCESS [  0.038 s]
	[INFO] Eagle::Server::Main ................................ SUCCESS [  0.157 s]
	[INFO] Eagle::App::Assembly ............................... SUCCESS [  0.002 s]
	[INFO] Eagle::App::JPM::SparkRunningJob ................... SUCCESS [  0.001 s]
	[INFO] Eagle::Server::Assembly ............................ SUCCESS [  0.001 s]
	[INFO] ------------------------------------------------------------------------
	[INFO] BUILD SUCCESS
	[INFO] ------------------------------------------------------------------------
	[INFO] Total time: 15.502 s
	[INFO] Finished at: 2016-11-21T12:52:04+08:00
	[INFO] Final Memory: 49M/1963M
	[INFO] ------------------------------------------------------------------------

Old name format is:

	[INFO] Apache Eagle Parent ................................ SUCCESS [  0.240 s]
	[INFO] eagle core ......................................... SUCCESS [  0.002 s]
	[INFO] eagle-data-process ................................. SUCCESS [  0.002 s]
	[INFO] eagle-common ....................................... SUCCESS [  0.002 s]
	[INFO] eagle-alert-parent ................................. SUCCESS [  0.002 s]
	[INFO] eagle-alert ........................................ SUCCESS [  0.003 s]
	[INFO] alert-common ....................................... SUCCESS [  0.025 s]
	[INFO] alert-engine ....................................... SUCCESS [  0.003 s]
	[INFO] alert-metadata-parent .............................. SUCCESS [  0.002 s]
	[INFO] alert-metadata ..................................... SUCCESS [  0.003 s]
	[INFO] alert-metadata-service ............................. SUCCESS [  0.002 s]
	[INFO] alert-coordinator .................................. SUCCESS [  0.002 s]
	[INFO] alert-assembly ..................................... SUCCESS [  0.004 s]
	[INFO] alert-devtools ..................................... SUCCESS [  0.001 s]
	[INFO] alert-service ...................................... SUCCESS [  0.002 s]
	[INFO] eagle-query-parent ................................. SUCCESS [  0.003 s]
	[INFO] eagle-antlr ........................................ SUCCESS [  0.002 s]
	[INFO] eagle-embed-parent ................................. SUCCESS [  0.001 s]
	[INFO] eagle-embed-hbase .................................. SUCCESS [  0.002 s]
	[INFO] eagle-entity-base .................................. SUCCESS [  0.003 s]
	[INFO] eagle-query-base ................................... SUCCESS [  0.002 s]
	[INFO] eagle-storage-base ................................. SUCCESS [  0.001 s]
	[INFO] eagle-embed-server ................................. SUCCESS [  0.004 s]
	[INFO] eagle-client-base .................................. SUCCESS [  0.002 s]
	[INFO] eagle-audit-base ................................... SUCCESS [  0.002 s]
	[INFO] eagle-storage-hbase ................................ SUCCESS [  0.002 s]
	[INFO] eagle-service-base ................................. SUCCESS [  0.002 s]
	[INFO] eagle-metadata ..................................... SUCCESS [  0.002 s]
	[INFO] eagle-metadata-base ................................ SUCCESS [  0.001 s]
	[INFO] eagle-alert-service ................................ SUCCESS [  0.002 s]
	[INFO] eagle-app-parent ................................... SUCCESS [  0.002 s]
	[INFO] eagle-app-base ..................................... SUCCESS [  0.003 s]
	[INFO] eagle-alert-app .................................... SUCCESS [  0.002 s]
	[INFO] eagle-storage-jdbc ................................. SUCCESS [  0.001 s]
	[INFO] eagle-metric ....................................... SUCCESS [  0.002 s]
	[INFO] eagle-metadata-jdbc ................................ SUCCESS [  0.001 s]
	[INFO] eagle-metadata-mongo ............................... SUCCESS [  0.002 s]
	[INFO] eagle-app-utils .................................... SUCCESS [  0.001 s]
	[INFO] eagle-security-parent .............................. SUCCESS [  0.001 s]
	[INFO] eagle-security-common .............................. SUCCESS [  0.002 s]
	[INFO] eagle-security-hdfs-auditlog ....................... SUCCESS [  0.001 s]
	[INFO] eagle-security-maprfs-auditlog ..................... SUCCESS [  0.002 s]
	[INFO] eagle-jpm-parent ................................... SUCCESS [  0.001 s]
	[INFO] eagle-jpm-util ..................................... SUCCESS [  0.001 s]
	[INFO] eagle-security-hive ................................ SUCCESS [  0.002 s]
	[INFO] eagle-security-hive-web ............................ SUCCESS [  0.001 s]
	[INFO] eagle-security-hdfs-web ............................ SUCCESS [  0.001 s]
	[INFO] eagle-security-maprfs-web .......................... SUCCESS [  0.002 s]
	[INFO] eagle-security-hdfs-authlog ........................ SUCCESS [  0.001 s]
	[INFO] eagle-security-hbase-auditlog ...................... SUCCESS [  0.001 s]
	[INFO] eagle-security-hbase-web ........................... SUCCESS [  0.001 s]
	[INFO] eagle-metric-collection ............................ SUCCESS [  0.002 s]
	[INFO] eagle-security-oozie-auditlog ...................... SUCCESS [  0.001 s]
	[INFO] eagle-security-oozie-web ........................... SUCCESS [  0.001 s]
	[INFO] eagle-external-parent .............................. SUCCESS [  0.002 s]
	[INFO] eagle-log4jkafka ................................... SUCCESS [  0.021 s]
	[INFO] eagle-kafka ........................................ SUCCESS [  0.001 s]
	[INFO] eagle-examples ..................................... SUCCESS [  0.001 s]
	[INFO] eagle-app-example .................................. SUCCESS [  0.001 s]
	[INFO] eagle-gc ........................................... SUCCESS [  0.002 s]
	[INFO] eagle-jpm-entity ................................... SUCCESS [  0.001 s]
	[INFO] eagle-jpm-mr-running ............................... SUCCESS [  0.002 s]
	[INFO] eagle-jpm-mr-history ............................... SUCCESS [  0.002 s]
	[INFO] eagle-jpm-web ...................................... SUCCESS [  0.002 s]
	[INFO] eagle-jpm-service .................................. SUCCESS [  0.001 s]
	[INFO] eagle-jpm-aggregation .............................. SUCCESS [  0.002 s]
	[INFO] eagle-jpm-spark-history ............................ SUCCESS [  0.001 s]
	[INFO] eagle-hadoop-queue ................................. SUCCESS [  0.001 s]
	[INFO] eagle-topology-check ............................... SUCCESS [  0.001 s]
	[INFO] eagle-topology-entity .............................. SUCCESS [  0.001 s]
	[INFO] eagle-topology-app ................................. SUCCESS [  0.002 s]
	[INFO] eagle-hadoop-metric ................................ SUCCESS [  0.002 s]
	[INFO] Eagle::Server ...................................... SUCCESS [  0.001 s]
	[INFO] eagle-topology-assembly ............................ SUCCESS [  0.002 s]
	[INFO] eagle-jpm-spark-running ............................ SUCCESS [  0.001 s]
	[INFO] eagle-server-assembly .............................. SUCCESS [  0.001 s]
	[INFO] ------------------------------------------------------------------------
	[INFO] BUILD SUCCESS
	[INFO] ------------------------------------------------------------------------
	[INFO] Total time: 2.044 s
	[INFO] Finished at: 2016-11-21T11:23:27+08:00
	[INFO] Final Memory: 49M/1963M
	[INFO] ------------------------------------------------------------------------

### CODE REVIEW

File: eagle-core/eagle-query/eagle-query-base/pom.xml [hunk 1]
```diff
@@ -28,6 +28,8 @@
 
     <artifactId>eagle-query-base</artifactId>
 
+    <name>Eagle::Core::Query::Base</name>
```
Comment (baibaichen) [inline eagle-core/eagle-query/eagle-query-base/pom.xml]: @haoch , how about <name>Eagle::Core::Query::_Query_Base</name>

Comment (haoch) [inline eagle-core/eagle-query/eagle-query-base/pom.xml]: Sounds good :-). Thanks.

**Comment by wujinhu** [comment]
LGTM

File: eagle-core/eagle-query/eagle-query-base/pom.xml (hunk 1 above)
Comment (haoch) [inline eagle-core/eagle-query/eagle-query-base/pom.xml]: @baibaichen Fixed as suggested.

**Pull request closed without merging**